### ISSUE TITLE
Set NOT_CRAN to true for the checks ?

### ISSUE THREAD

**Issue by gaborcsardi** [r-lib/revdepcheck#23]
Maybe optionally?

**Comment by hadley** [comment]
I don't think we want to do this, just so we're as similar to CRAN as possible

**Comment by jimhester** [comment]
So callr has 
https://github.com/r-lib/callr/blob/2dffbbe502c7aba84fc0785add4a1e588f88b03c/R/rcmd.R#L106

which basically always sets `NOT_CRAN=true` unless it is already set, which I believe is causing revdepcheck to always have `NOT_CRAN=true` set when running revdepchecks. I am pretty sure I have purged `NOT_CRAN` settings in all of my environment configurations and it is still ends up being set when I run `revdepcheck::revdep_check()`

**Comment by gaborcsardi** [comment]
`callr` probably should not touch `NOT_CRAN` at all, no? @hadley ?

**Comment by jimhester** [comment]
I think not touching it is probably best.

In the meantime I can successfully work around it with
```r
withr::with_envvar(c(NOT_CRAN="false"), revdepcheck::revdep_check())
```

**Comment by gaborcsardi** [comment]
> Hmmmm, I'm not sure. We are asserting that the code is not running on CRAN, which is true.

In `callr` not aways, actually, because we do use `callr::r` for many things now, and some of these do run on CRAN.

**Comment by hadley** [comment]
Ok, that's true. We should be setting `NOT_CRAN` at a higher level.
